### DISCUSSION TITLE
Module '"react-hook-form"' has no exported member 'UseFormMethods'

### DISCUSSION THREAD

**Discussion by alenseferovic1** [react-hook-form/react-hook-form#5145] [General] [answered]
I'm trying to update react-hook-form from v6 to v7 and I can't access to "UseFormMethods" anymore. This is my import:
 import { UseFormMethods } from 'react-hook-form';

**Comment by bluebill1049** [comment, marked as answer]
remove your node_moduel and do a fresh install.

**Reply by alenseferovic1** [reply]
I have already did that when I have upgraded version to latest I have deleted node_modules and install them again

**Reply by lamhungypl** [reply]
should be `UseFormReturn` in v7

**Reply by bluebill1049** [reply]
ty! let me fix the doc.

**Reply by licatajustin** [reply]
@bluebill1049 this might be something great to add to the [Migration Docs](https://react-hook-form.com/migrate-v6-to-v7) too 😄

**Reply by bluebill1049** [reply]
sure @licatajustin I will add a section to refer type names against the ts page in the doc.

**Reply by licatajustin** [reply]
Huzzah! Thanks for all you do on this project, @bluebill1049!
